### PR TITLE
Add python csv to json converter

### PR DESCRIPTION
**Pull request by helias22**
## Pull Request Summary
Added very simple csv to json converter that we can use for our transcripts.

### CODE REVIEW

**Review by WeissSa — CHANGES_REQUESTED** [review]
Can we put this in a folder (i.e. examples/csv_to_json.py)

**Review by WeissSa — COMMENTED** [review]
Sorry one more thing. Maybe add as a comment at the top of the file what CSV this is expecting and how to run it.

File: back-end/dao/intentdao.js [hunk 1]
```diff
@@ -8,7 +8,9 @@ export default class IntentDao extends IntentInterface{
 
   async getIntent(query = {}) {
     try {
+      console.log(query)
```
Comment (williamyao27) [inline back-end/dao/intentdao.js:11]: are these supposed to be here?

**Review by williamyao27 — APPROVED** [review]
no QA (as in I did not try the python script on a csv) but code LGTM.

just a bit confused on the console logs in intentdao.

File: back-end/dao/intentdao.js [hunk 2]
```diff
@@ -8,7 +8,9 @@ export default class IntentDao extends IntentInterface{
...
       const intentList = await Intent.find(query).exec();
+      console.log(intentList)
```
Comment (WeissSa) [inline back-end/dao/intentdao.js:13]: +1 we should delete these

**Review by WeissSa — APPROVED** [review]
Just remove the logs and we are rolling

File: back-end/dao/intentdao.js (hunk 1 above)
Comment (helias22) [reply back-end/dao/intentdao.js:11]: absolutely not :))

File: back-end/dao/intentdao.js (hunk 2 above)
Comment (helias22) [reply back-end/dao/intentdao.js:13]: My bad :))

**Pull request merged**